Src/FluentAssertions/Equivalency/SelfReferenceEquivalencyAssertionOptions.cs

```diff
@@ -576,7 +576,8 @@ public TSelf ComparingRecordsByValue()
         }
 
         /// <summary>
-        /// Ensures records by default are compared by value instead of their members.
+        /// Ensures records by default are compared by its members even though it may override
```

Comment:
Doesn't a record _always_ override `object.Equals`?

My grammar tells me that `its` and `it` should `their` and `they` as `records` is plural.
```
Ensures records by default are compared by their members 
even though they override the <see cref="object.Equals(object)" /> method.
```

Reply:
maybe. that's _me_ doing copy/paste from the comments below and trying to "align them". I didn't think about it much past that point. 

but was my initial suspicion correct that the existing comment is incorrect?

Reply:
You can always count on @jnyrup for a keen eye for detail ;-)

Reply:
my next question is figuring out how to edit the file. 
(I made the PR right in github web) :-D

Reply:
found it!
updated.